README.md

```diff
@@ -1,82 +1,82 @@
-=Wagn: how pioneers roll
+#Wagn: how pioneers roll
 
-==Basics
+[![Build Status](https://semaphoreapp.com/api/v1/projects/0d61c1f9-5ef0-4c5c-89c6-7664e247e4be/294670/badge.png)](https://semaphoreapp.com/ethan/wagn)
```

Comment:
I think this is wrong.  README should direct them to INSTALL or README_INSTALL, if at all.  The REAME and in particular the badges isn't part of the gem, but the repo.  I don't think they'll even see it if they install the gem.  You can put another README (INSTALL might be better) in the wagn gem for that purpose.  We may need a more general overhaul of these doc files.

Like I said, I'm fine if you don't want this at all, but I don't think you have good arguments for why it is a bad idea.  It may also point out we have deficiencies here with nothing to do with the badge or markdown.